composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/screens/BookReservationScreen.kt

```diff
@@ -104,8 +106,8 @@ fun BookReservationLayout(
 
             UpcomingReservation(
                 modifier = Modifier.padding(bottom = 24.dp),
-                startTime = LocalDateTime.parse(state.reservation?.startTime ?: "null"),
-                endTime = LocalDateTime.parse(state.reservation?.endTime ?: "null"),
+                startTime = state.reservation?.startTime?.toLocalDateTime() ?: Clock.System.now().toLocalDateTime(),
```

Comment:
Je hoeft niet elke keer "Clock.system.now()" te doen, ik heb ergens een  variable "now" gemaakt die hetzelfde is, is een stuk korter.